.github/pull_request_template.md

```diff
@@ -0,0 +1,43 @@
+<!--
+**Thanks for contributing to Fury.**
+
+**If this is your first time opening a PR on fury, you can refer to [CONTRIBUTING.md](https://github.com/apache/incubator-fury/blob/main/CONTRIBUTING.md).**
+
+Contribution Checklist
+
+    - The **Apache Fury (incubating)** community has restrictions on the naming of pr titles. You can also find instructions in [CONTRIBUTING.md](https://github.com/apache/incubator-fury/blob/main/CONTRIBUTING.md).
+
+    - Fury has a strong focus on performance. If the PR you submit will have an impact on performance, please benchmark it first and provide the benchmark result here.
+-->
+
+## What does this PR do?
+
+<!-- Describe the purpose of this PR. -->
+
+
+## Related issues
+
+<!--
+Is there any related issue? Please attach here.
+
+- #xxxx0
+- #xxxx1
+- #xxxx2
+-->
+
+
+## Does this PR introduce _any_ user-facing change?
```

Comment:
```suggestion
## Does this PR introduce any user-facing change?
```

Reply:
_any_(\_any\_) is in italics to emphasize importance, which I think is necessary

Reply:
I prefer not emphasize it, the items list here are all important